cookbooks/bunge_et_al_mantle_convection/bunge_et_al.prm

```diff
@@ -0,0 +1,93 @@
+# Setup for convection in a quarter of a 2d shell.
```

Comment:
quarter shell or a full shell?

Reply:
I don't think you addressed this yet?

Reply:
I don't get it.. I push a commit yesterday evening with all corrections (except data files location)